ui/app/AppLayouts/Chat/ChatLayout.qml

```diff
@@ -108,6 +147,20 @@ StackLayout {
             onOpenAppSearch: {
                 root.openAppSearch()
             }
+            onRevealAddressClicked: {
+                Global.openPopup(communityIntroDialogPopup, {
+                    communityId: root.sectionItemModel.id,
+                    isInvitationPending: root.rootStore.isCommunityRequestPending(root.sectionItemModel.id),
+                    name: root.sectionItemModel.name,
+                    introMessage: root.sectionItemModel.introMessage,
+                    imageSrc: root.sectionItemModel.image,
+                    accessType: root.sectionItemModel.access
+                })
+            }
+            onInvitationPendingClicked: {
+                root.rootStore.cancelPendingRequest(root.sectionItemModel.id)
+                chatView.isInvitationPending = root.rootStore.isCommunityRequestPending(root.sectionItemModel.id)
```

Comment:
This will break the binding that you defined above on line 139

Reply:
I think this is fine. It has been like that previously as well, and then there's a connection on the community module that updates this property again when a request state change happens